utils/location.go

```diff
@@ -163,7 +163,12 @@ func (h *LocationHierarchy) FindByName(name string, level LocationLevel, parent
 
 // FindByPath looks for a location in the hierarchy with the given path
 func (h *LocationHierarchy) FindByPath(path string) *Location {
-	return h.pathLookup.lookup(strings.ToLower(path))
+	tokens := strings.Split(path, ">")
+	for i := range tokens {
+		tokens[i] = strings.TrimSpace(tokens[i])
+	}
+	normalizedPath := strings.ToLower(strings.TrimRight(strings.Join(tokens, "\x20>\x20"), "."))
```

Comment:
don't have to use `\x20` here instead of just a space - i just put that in the in the issue cuz it isn't obvious that ` > ` has spaces